### PR TITLE
Fix delete global ip address

### PR DESCRIPTION
**Pull request by kazyshr**
I made some fixes for delete method.
- Change OK status code to 200

- Handle response body

- Update doc

- Fix typo of test function name

### CODE REVIEW

File: fic/eri/v1/routers/components/nat_global_ip_address_sets/doc.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ Example to List Global IP Address Sets
```
Comment (naoshark) [inline fic/eri/v1/routers/components/nat_global_ip_address_sets/doc.go]: `listOpts := nat_global_ip_address_sets.ListOpts{}` nit: no need of this

Comment (kazyshr) [reply fic/eri/v1/routers/components/nat_global_ip_address_sets/doc.go]: Removed

**Review by naoshark — APPROVED** [review]
LGTM

**Review by shimisho — APPROVED** [review]
LGTM.
The OK status code for the Create API is also 200, do we need to make the same corrections?

**Comment by kazyshr** [comment]
> LGTM.
> The OK status code for the Create API is also 200, do we need to make the same corrections?

Thanks for the comment. I corrected it.

**Pull request merged**